TestingTravis/TestClass.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  testClass.swift
+//  TestingTravis
+//
+//  Created by Zomato on 14/03/19.
+//  Copyright © 2019 tester. All rights reserved.
+//
+
+import Foundation
+
+class TestClass: NSObject {
+    
+    var forceUnwrapMe: String?
+    
+    override init() {
+        print(forceUnwrapMe!)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/2ab967952d0734b15797e4d70bbe1b3c3fb02e82/TestingTravis/TestClass.swift#L16"></span>Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>TestClass.swift:16</code></td>
    </tr>
  </tbody>
</table>